### PR TITLE
feat: Add language errors and vadlidateDsl

### PR DESCRIPTION
**Pull request by d-jeffery**
## Description

* Git submodule used to add language as a dependency
* `checkDSL` & `CMD` commands updated to use `validateDsl` from `language`
* Removed outdated test suite, as all but the 1.0 specific tests have been migrated
* Fixed eslint duplicate config issue
* Excluded `language` from prettier & madge as a temporary fix to checking failures for submodule

## References

- closes https://github.com/openfga/syntax-transformer/issues/141

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   "author": "OpenFGA",
   "dependencies": {
     "@openfga/sdk": "^0.2.7",
+    "@openfga/language": "./src/language/pkg/js",
```
Comment (adriantam) [inline package.json:36]: Curious why we do this - if we intend to have language as part of src, can we not just import them without specifying it as dependencies here?

Comment (d-jeffery) [reply package.json:36]: Yep, thats fine. I just thought the change could be made a little more seamlessly for here: https://github.com/openfga/syntax-transformer/pull/152/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R10

But happy to take it out.

Comment (d-jeffery) [reply package.json:36]: After change, rediscovered this was an easy way of not having to declare a top level dependency on `antlr`

File: src/index.ts [hunk 2]
```diff
@@ -7,6 +7,8 @@ import * as syntaxHighlighters from "./syntax-highlighters";
 import * as theming from "./theme";
 import * as graphBuilder from "./utilities/graphs";
 import sampleAuthorizationModels from "./samples";
+// Imported like a module to avoid including antlr in the top level dependencies
+import * as validateDsl from "@openfga/language";
```
Comment (rhamzeh) [inline src/index.ts:11]:
```suggestion
import * as language from "@openfga/language";
```

**Review by rhamzeh — CHANGES_REQUESTED** [review]

**Review by rhamzeh — APPROVED** [review]

**Pull request merged**